src/components/FormSubmit/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import lodashGet from 'lodash/get';
+import {View} from 'react-native';
+import {propTypes, defaultProps} from './formSubmitPropTypes';
+
+class FormSubmit extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.submitForm = this.submitForm.bind(this);
+    }
+
+    submitForm(event) {
```

Comment:
Why are we calling this submitForm? There is no form in the render. Missing jSDoc.